proxy/util/cloudsqlutil.go

```diff
@@ -43,3 +46,19 @@ func SplitName(instance string) (project, region, name string) {
 		return spl[0], spl[1], spl[2]
 	}
 }
+
+// Validate verifies that instances are in the expected format and include
```

Comment:
Looks like we accidentally pulled in this change during the rebase. This can be deleted.